app/server/utils/user.py

```diff
@@ -200,13 +200,14 @@ def create_transfer_account_user(
 
     if organisation:
         user.organisations.append(organisation)
+        user.default_organisation = organisation
```

Comment:
was this here before?

Reply:
So I may have done the cherry pickin thing wrong, there were so many breakages from the old owen/ussd_features branch, so I mirrored master's present state into my branch to resolve those issues.